### DISCUSSION TITLE
Cant delete calibrations

### DISCUSSION THREAD

**Discussion by rozwand** [NightscoutFoundation/xDrip#2890] [Q&A]
I am using a oop2 + xdrip for libre2 sensor was working for 4 days with bluetooth. And then it stopped working. Do i used NFC to read data and its was working fine for 20 min. Now its not sending data by bluetooth or NFC. I restarted calibrations but they are still there.

![Screenshot_20230616_082502_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/69338080/984cfacb-3e1f-4de0-a7f3-bb7260a62ba7)
![Screenshot_20230616_082509_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/69338080/5e3d209c-cc7a-46a9-aec9-f25f00914076)
![Screenshot_20230616_083637_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/69338080/a8da7fee-f6d6-41ec-9494-14e86ade0d60)
![Uploading Screenshot_20230616_085228_OOPalgorithm2.jpg…]()

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
Please use "No calibration" for now (to test), this should reset calibrations (the same can be achieved like this: https://xdrip.readthedocs.io/en/latest/calibrate/calibrate/#reset-all-calibrations). Your sensor return BG=0 that creates problems to xDrip+ https://xdrip.readthedocs.io/en/latest/use/misc/#oop2
